client/src/components/Project/project.jsx

```diff
@@ -0,0 +1,48 @@
+import { useState, prompt} from "react";
+import "./project.css";
+import sproject from "../../assets/images/sproject.png";
+
+function Project() {
+  const [project, setProject] = useState([
```

Comment:
Dont use static data, you should fetch your api in a useEffect and update your state with the data returned by your api